db/jksndb.sql

```diff
@@ -0,0 +1,217 @@
+-- Schema jksndb
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `jksndb` DEFAULT CHARACTER SET utf8 ;
+USE `jksndb` ;
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`company`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`company` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(45) NULL,
+  `Address1` VARCHAR(45) NULL,
+  `Address2` VARCHAR(45) NULL,
+  `cell_phone` VARCHAR(45) NULL,
+  `office_phone` VARCHAR(45) NULL,
+  `City` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`role`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`role` (
+  `id` INT NOT NULL,
+  `role` INT NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`user`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`user` (
+  `username` (16) NOT NULL,
+  `email` (255) NULL,
+  `password` (32) NOT NULL,
+  `create_time`  NULL DEFAULT CURRENT_TIMESTAMP,
+  `id` INT NOT NULL,
+  `wechat_id` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`company` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`role` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION);
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`vendor`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`vendor` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(45) NULL,
+  `city` VARCHAR(45) NULL,
+  `cell_phone` VARCHAR(45) NULL,
+  `office_phone` VARCHAR(45) NULL,
+  `address1` VARCHAR(45) NULL,
+  `address2` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`category`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`category` (
+  `id` INT NOT NULL,
+  `discription` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`equip`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `jksndb`.`equip` (
+  `id` INT NOT NULL,
+  `model` VARCHAR(45) NULL,
+  `sn` VARCHAR(45) NULL,
+  `discription` VARCHAR(45) NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`vendor` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION,
+  CONSTRAINT `id`
+    FOREIGN KEY ()
+    REFERENCES `jksndb`.`category` ()
+    ON DELETE NO ACTION
+    ON UPDATE NO ACTION)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `jksndb`.`order`
```

Comment:
we need a filed to identify this order belongs to whom. call 'buyer'  or 'owner'? 
I.e.  The admin  may help end user make one order.   In this case,  shall need 2 field to seprate?